gips/0066-graph-horizon.md

```diff
@@ -0,0 +1,348 @@
+---
+GIP: 0066
+Title: Introducing Graph Horizon - A data services protocol
+Authors: Tomás Migone <tomas@edgeandnode.com>, Pablo Carranza Vélez <pablo@edgeandnode.com>, Miguel de Elias <miguel@edgeandnode.com>
+Created: 2024-05-03
+Updated: 2024-08-16
+Stage: Candidate
+Discussions-To: https://forum.thegraph.com/t/<xxxx>
+Category: Protocol Logic, Protocol Interfaces, Economic Parameters
+Depends-On: GIP-0068
+Implementations: https://github.com/graphprotocol/contracts/pull/944
+Audits:
+---
+
+# Abstract
+
+In this proposal we present our design for Graph Horizon, a data services protocol which is the next evolution of The Graph. We also introduce the Data Service framework, a library and set of guidelines to develop data services using Graph Horizon primitives; and the Subgraph Service, the first data service built to support the current use case of The Graph, indexing subgraphs. 
+
+These changes require a complete redesign, upgrading most of the protocol contracts. A companion GIP-0068 will cover the details of the Subgraph Service. Note however that both GIPs should be treated as a unit, meaning one cannot be implemented without the other.
+
+Finally, the Graph Council has expressed support for the the [brownfield approach](https://forum.thegraph.com/t/graph-horizon-explained-a-proposal-for-the-evolution-of-the-protocol/5169#two-ways-to-get-there-brownfield-or-greenfield-11) so we propose a way of safely transitioning from the current version of the protocol to Graph Horizon. 
+
+# Motivation
+
+The Graph launched over two years ago with the objective of being an efficient and censorship-resistant protocol that can scale to organize all of the world’s public data and information. Since then there have been significant changes in the ecosystem and we’ve learned from conversations with many users and contributors. [GIP-0042](https://forum.thegraph.com/t/gip-0042-a-world-of-data-services/3761) introduced the concept of data services, recognizing that other approaches besides subgraphs are possible. [Graph Horizon](https://forum.thegraph.com/t/graph-horizon-explained-a-proposal-for-the-evolution-of-the-protocol/5169) expands on this idea, presenting a modular, composable and permissionless staking and payments protocol that can be leveraged by different data service implementations to offer a wide range of solutions to satisfy all data needs.
```

Comment:
> The Graph launched over two years ago

If we're referring to the network (mainnet), shouldn't we say 4 years, or reference the actual date that is December 2020?